java/app/grapheneos/setupwizard/view/activity/MigrationActivity.kt

```diff
@@ -17,6 +18,19 @@ class MigrationActivity : SetupWizardActivity(
         private const val TAG = "MigrationActivity"
     }
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        // check if seedvault exists
+        val intent = Intent(getString(R.string.restore_action))
+        if (intent.resolveActivity(packageManager) == null) {
+            // should only run if seedvault is missing
+            Log.d(TAG, "Seedvault is gone!")
+            // after this action is executed users are unable to go back to steps before this one (that is why i had the alertbox show)
+            SetupWizard.next(this)
```

Comment:
this will cause crash as a call to `super.onCreate(..)` is missing.
also this activity needs to finish itself before moving to next step (that fixes your line 27 issue).

Reply:
might have fixed it

Reply:
@jiteshsingh